files/en-us/web/accessibility/accessibility_colon__what_users_can_to_to_browse_safely/index.html

```diff
@@ -33,7 +33,7 @@ <h3 id="Use_modern_browsers._Learn_personalization_and_accessibility_settings">U
 
 <p>Chrome 74 added support for <strong><code>reduced-motion-query</code></strong>, it has even added support to detect whether the user has requested a reduced motion experience. The user would access this through an accessibility interface, as seen below.</p>
 
-<p><img alt="Screen shot showing how to turn off animations in Windows10" src="https://mdn.mozillademos.org/files/16662/android-remove-animations.png" style="height: 493px; width: 472px;"></p>
+<p><img alt="Screen shot showing how to turn off animations in Windows10" src="android-remove-animations.png"></p>
```

Comment:
```suggestion
<p><img alt="Screen shot showing how to turn off animations in Windows 10." src="android-remove-animations.png"></p>
```

Little tweaks for better pronunciation.